pkg/spacebinding/spacebindinglister_test.md

```diff
@@ -0,0 +1,32 @@
+## SpaceBinding lister test 
+
+The spacebinding lister test verifies that `ListForSpace` function returns the expected spacebindings for a given space resource.
+
+At a high level, we want to make sure that given a specific space, the function covers the following requirements:
+
+- we get spcebindings resources for users that have access to that space
+- we want to make sure the spacebindings are being inherited from the parent spaces
+- spacebindings on the sub-space will override those on the parent space for the same MUR
+- users can have different roles (or no access at all) on the various spaces in the hierarchy
```

Comment:
one user should have one single role in a particular Space

Reply:
hmm yeah, let me change that last sentence a bit. thanks!

Reply:
changed in https://github.com/codeready-toolchain/toolchain-common/pull/329/commits/6f1d0354d896c222992c4c482fc0b80cfab19427